tests/framework/rbac/PhpManagerTest.php

```
@@ -6,161 +6,325 @@
-    /**
-     * {@inheritdoc}
-     */
-    protected function createManager()
-    {
-        return new ExposedPhpManager([
+        protected function removeDataFiles()
+        {
+            @unlink($this->getItemFile());
+            @unlink($this->getAssignmentFile());
+            @unlink($this->getRuleFile());
+        }
```

PHPMD (2.15.0)

[error] Remove error control operator '@' on line 76. (undefined)

(ErrorControlOperator)